### PR TITLE
Html file for home has been created with Tags and IDS

### PR DESCRIPTION
**Pull request by Mehmar-MohammadMubeen**

### CODE REVIEW

File: JS/home.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+              </article>
+              <article class="box-container" id="classesCard"style="height: 200px; width: 300px">
+                  <h2>5</h2>
+                  <p>number of classes</p>
+              </article>
+          </section>
+      `;
+    }
+    function addHeading() {
+      const heading = document.createElement("h2");
```
Comment (mehmet-hicoders) [inline JS/home.js:65]: Do not use DOM manipulation APIs, only innerHTML

**Review by mehmet-hicoders — CHANGES_REQUESTED** [review]

**Comment by mehmet-hicoders** [comment]
you can merge it

**Comment by Mehmar-MohammadMubeen** [comment]
Ok Sir

Mohammad Mubeen Mehmar
________________________________

File: JS/home.js (hunk 1 above)
Comment (Mehmar-MohammadMubeen) [reply JS/home.js:65]: Ok sir

**Pull request merged**